test/offsetarray.jl

```diff
@@ -303,6 +310,10 @@ map!(+, dest, am, am)
 @test dest[1,8] == 4
 @test dest[1,9] == -2
 
+am = map(identity, a)
```

Comment:
The whole file is inside a `let`: https://github.com/JuliaLang/julia/blob/c10667fda172138a376be4c72d7db3bd4b4f323e/test/offsetarray.jl#L103